be/src/olap/olap_common.h

```diff
@@ -143,7 +142,8 @@ enum FieldType {
     OLAP_FIELD_TYPE_HLL = 23,
     OLAP_FIELD_TYPE_BOOL = 24,
     OLAP_FIELD_TYPE_OBJECT = 25,
-    OLAP_FIELD_TYPE_STRING = 26
+    OLAP_FIELD_TYPE_STRING = 26,
+    OLAP_FIELD_TYPE_SIZE = 27 // Size of FieldType
```

Comment:
And comment to notice that this must be the last field of `FieldType`

Reply:
> And comment to notice that this must be the last field of `FieldType`

done